app/views/coronavirus_form/have_somewhere_to_live.html.erb

```diff
@@ -1 +1,4 @@
-<%= render partial: "question_radio_buttons", locals: {question: "have_somewhere_to_live"} %>
+<%= render partial: "question_radio_buttons", locals: {
+  question: controller_name,
+  group: group,
```

Comment:
Where did you come from? I think the rebase went a bit rogue.

Reply:
This is from #30 - updated to use the same variables as the other questions.